tools/mo/requirements_mxnet.txt

```diff
@@ -1,5 +1,5 @@
 mxnet~=1.2.0; sys_platform == 'win32'
-mxnet~=1.7.0.post2; sys_platform != 'win32'
+mxnet>=1.7.0.post2; sys_platform != 'win32'
```

Comment:
we should always have upper-bound in extras it will be a version with which our validation is performed adn we gives some warranty that it will work on the customer machine.

Just boost upper bound up to the latests top version